### PR TITLE
Fix broken scoring for storing groceries.

### PR DESCRIPTION
**Pull request by LeroyR**
## Description

Closes issue #855

Changes proposed in this pull request:
- DeuxEx rules now mandate 'clear indication' so we should remove that part
- removed duplicate penalty item

I tried to somewhat match penalties to the scores but i am not sure of the original intention

### CODE REVIEW

**Review by hawkina — DISMISSED** [review]
looks good to me. Thank you.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

**Review by PetervDooren — CHANGES_REQUESTED** [review]
This PR is combining two things.
1- fixing the duplicate points for deus ex machina
2 - adjusting the points for deus ex machina. mostly with regards to deus ex machinas for humans doing tasks for the robot.

While 1 is fairly straightforward. 2 is quite a change and would require looking at more challenges. 
I would approve a simple change in 1. but not this combination.

File: scoresheets/StoringGroceries.tex [hunk 1]
```diff
@@ -19,11 +19,9 @@
 	\penaltyitem[5]{-60}{Storing an object without categorizing it correctly}
 
 	\scoreheading{Deus Ex Machina Penalties}
-	\penaltyitem[5]{-15}{A human handing an object over to the robot (the object is clearly indicated by the robot)}
 	\penaltyitem[5]{-30}{A human handing an object over to the robot}
-	\penaltyitem[5]{-45}{A human placing an object in the cabinet at a location clearly indicated by the robot}
-	\penaltyitem[5]{-90}{A human placing an object in the cabinet}
-	\penaltyitem[5]{-30}{A human placing an object in the cabinet at a location clearly indicated by the robot}
```
[the remark this answers is not in the document]
Comment (LeroyR) [reply scoresheets/StoringGroceries.tex:26]: The Deux ex Machina rules state "Detailed information to identify objects for picking and placing" so the human can only assist if the robot indicates a location.

File: scoresheets/StoringGroceries.tex [hunk 2]
```diff
@@ -19,11 +19,9 @@
...
+	\penaltyitem[5]{-10}{A human placing an object in the cabinet}
+	\penaltyitem[5]{-30}{A human placing an object in the cabinet next to similar objects}
```
[the remark this answers is not in the document]
Comment (LeroyR) [reply scoresheets/StoringGroceries.tex:24]: The "next to a similar" object penalty is to reduce "Placing an object next to similar objects on the cabinet" score.
Theres two possible deux ex machinas during placing:

1. Robot requests to place the object at the wrong location
2. Robot requests to place the object at the correct location

Other help requests e.g. "place it on the shelf" have to be ignored.

I understand the rules such that all applicable penalties will be applied thus:

1. "Placing an object in the cabinet" +15, "A human placing an object in the cabinet" -10 = 5 points
2. "Placing an object in the cabinet" +15, "Placing an object next to similar objects on the cabinet" +50, "A human placing an object in the cabinet" -10 "A human placing an object in the cabinet next to similar objects" - 30 = 25 Points

[the remark this answers is not in the document]
Comment (johaq) [reply scoresheets/StoringGroceries.tex:24]: Yes, I think @LeroyR is correct. With the changes made in #838, generic requests by the robot will be ignored thus mentioning them in the score section is just confusing for teams. Therefore they should be ignored. This is change already implemented in the release by #838 just made more consistent by this PR

**Comment by johaq** [comment]
> This PR is combining two things. 1- fixing the duplicate points for deus ex machina 2 - adjusting the points for deus ex machina. mostly with regards to deus ex machinas for humans doing tasks for the robot.
> 
> While 1 is fairly straightforward. 2 is quite a change and would require looking at more challenges. I would approve a simple change in 1. but not this combination.


Adjustment of 2 was discussed and approved in the last TC meeting. These changes should go through. It does not make sense for the robot to get a lot of points for something a human did.

**Review by johaq — CHANGES_REQUESTED** [review]
Imo it should be:
- A human handing an object over to the robot -50 (skips Picking up an object for transportation to the cabinet 50)
- A human placing an object in the cabinet -15 (skips Placing an object in the cabinet 15)
- A human placing an object in the cabinet next to similar objects -50 (skips Placing an object next to similar objects on the cabinet 50)
- A human pointing at a target location -30/-25 (this is robot placing but skipping the recognition part which should give -35 but i think the human pointing detection deserves some points here, therefore -30 or -25)

File: scoresheets/StoringGroceries.tex [hunk 3]
```diff
@@ -19,12 +19,10 @@
...
 	\scoreheading{Deus Ex Machina Penalties}
-	\penaltyitem[5]{-15}{A human handing an object over to the robot (the object is clearly indicated by the robot)}
-	\penaltyitem[5]{-30}{A human handing an object over to the robot}
-	\penaltyitem[5]{-45}{A human placing an object in the cabinet at a location clearly indicated by the robot}
-	\penaltyitem[5]{-90}{A human placing an object in the cabinet}
-	\penaltyitem[5]{-30}{A human placing an object in the cabinet at a location clearly indicated by the robot}
-	\penaltyitem[5]{-45}{A human pointing at a target location}
+	\penaltyitem[5]{-50}{A human handing an object over to the robot}
+	\penaltyitem[5]{-15}{A human placing an object in the cabinet}
+	\penaltyitem[5]{-30}{A human placing an object in the cabinet next to similar objects}
```
Comment (YuqianJiang) [inline scoresheets/StoringGroceries.tex:24]: "A human placing an object in the cabinet next to similar objects" is -30 (robot placing is +50) which is different from what @johaq suggested. Is it intended?

Comment (johaq) [reply scoresheets/StoringGroceries.tex:24]: Yes should be -50 imo.

**Comment by LeroyR** [comment]
See #862 for updates

**Pull request closed without merging**